apps/src/experiments.js

```diff
@@ -1,13 +1,18 @@
 /**
  * This module contains logic for tracking various experiments. Experiments
- * can be enabled/disabled one of two ways
- * (1) enable: localStorage.set('experiments-experimentName', true)
- *     disable: localStorage.removeItem('experiments-experimentName')
- * (2) Add a query param, i.e. http://foo.com?experimentName=true or
- *     http://foo.com?experimentName=false
- * The latter approach ends up toggling the localStorage state.
+ * can be enabled/disabled using query parameters:
+ *   enable:  http://foo.com/?enableExperiment=experimentOne,experimentTwo
+ *   disable: http://foo.com/?disableExperiment=experimentOne,experimentTwo
+ * Experiment state is persisted across page loads using local storage.
  */
+
+var queryString = require('query-string');
+
 var experiments = module.exports;
+// TODO(pcardune): remove OLD_KEY_WHITELIST when whitelisted experiments have
+// shipped
+var OLD_KEY_WHITELIST = ['topInstructions', 'runModeIndicators'];
```

Comment:
could get rid of `runModeIndicators` which isn't actually used anywhere yet (but was going to be the next key i did use)